ts/src/base/Exchange.ts

```diff
@@ -3723,10 +3723,18 @@ export default class Exchange {
             precision = this.safeValue (networkItem, 'precision', precision);
         }
         if (precision === undefined) {
-            return fee;
+            return this.forceString (fee);
```

Comment:
this change is in base. it affects **only few exchanegs** where currency precisions are not available from `fetchCurrencies`.
for those exchanges, we need to return `string` as in `else` clause through `decimalToPrecision` we return `string`. 
so, currently we have a bug as in few exchanges we are returning `numeric` instead of `string`